src/components/TeamProfilePreview/TeamProfilePreview.tsx

```diff
@@ -60,7 +69,35 @@ const TeamProfilePreview = ({ groupId }: UserProps): JSX.Element => {
 
                         <NarrowSection>
                             <TransferGroupForm group={group} />
-                            <InlineTrigger icon={<IconBinOutline size="xs" />} text={tr('Archive group')} disabled />
+
+                            <ArchiveGroupModal
+                                visible={archiveGroupModalVisibility.value}
+                                group={group}
+                                onClose={archiveGroupModalVisibility.setFalse}
+                            />
+                            {user?.role === UserRole.ADMIN &&
+                                (childrenQuery.data?.length ? (
+                                    <>
+                                        <div ref={ref}>
```

Comment:
Why not put ref on InlineTrigger?

Reply:
If InlineTrigger has a disabled prop, then popup is not visible